test/e2e/network/netpol/test_helper.go

```diff
@@ -67,7 +68,7 @@ func UpdatePolicy(k8s *kubeManager, policy *networkingv1.NetworkPolicy, namespac
 
 // waitForHTTPServers waits for all webservers to be up, on all protocols sent in the input,  and then validates them using the same probe logic as the rest of the suite.
 func waitForHTTPServers(k *kubeManager, model *Model) error {
-	const maxTries = 10
+	const maxTries = 20
```

Comment:
see comment above on max tries, this is only needed in case of extreme congestion, but it wont slow things down from what were seeing in measurements.

Reply:
@rajaskakodkar , can you make the call on wether or not we stil need this?

Reply:
I think we can revert this